chef-server-stats/chef-server-stats

```diff
@@ -295,6 +316,7 @@ output.merge!(get_authz_stats(services))
 output.merge!(get_redis_stats(services))
 output.merge!(get_server_status)
 output.merge!(get_folsom_metrics(chef_server_12))
+output.merge!(get_pooler_metrics(chef_server_12))
```

Comment:
Great point. We implemented bounded queues at 12.2 so that's where we'll start returning data, I think we will just get no JSON back in that circumstance but let me give it a test today and comment back here once I do.